rancher/v1.3/en/cattle/rancher-compose/index.md

```diff
@@ -24,7 +24,7 @@ You can also check the [releases page for Rancher Compose](https://github.com/ra
 
 ### Setting up Rancher Compose with Rancher Server
 
-To enable Rancher Compose to launch services in a Rancher instance, you'll need to set environment variables or pass these variables as an option in the Rancher Compose command. The environment variables that are required are`RANCHER_URL`, `RANCHER_ACCESS_KEY`, and `RANCHER_SECRET_KEY`. The access key and secret key will be an [environment API key]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/api/api-keys/). Environment API keys can be created in **API** -> **Advanced Options**.
+To enable Rancher Compose to launch services in a Rancher instance, you'll need to set environment variables or pass these variables as an option in the Rancher Compose command. The environment variables that are required are`RANCHER_URL`, `RANCHER_ACCESS_KEY`, and `RANCHER_SECRET_KEY`. The access key and secret key will be an [environment API key]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/api/v1/api-keys/). Environment API keys can be created in **API** -> **Advanced Options**.
 
```

Comment:
I'd actually prefer it to point to `v2-beta` instead of `v1`. Would you be able to update the PR?